tools/acl_compressor/sources/main.cpp

```diff
@@ -374,24 +353,53 @@ int main(int argc, char** argv)
 	run_unit_tests();
 #endif
 
-	const Options options = parse_options(argc, argv);
+	Options options;
+
+	if (!parse_options(argc, argv, options))
+	{
+		return -1;
+	}
+
+	if (options.input_filename == nullptr)
```

Comment:
shouldn't parse_options do this sort of validation  if it returns a bool anyway?